priv/templates/phx.gen.live/live_helpers.ex

```diff
@@ -4,6 +4,26 @@ defmodule <%= inspect context.web_module %>.LiveHelpers do
 
   alias Phoenix.LiveView.JS
 
+  @doc """
+  Renders a flash message.
+
+  The rendered flash receives a `:type` that will be used to define
+  proper classes to the element, and a `:message` which will be the
+  inner HTML.
+  If message is empty, the element is not rendered.
+
+  ## Examples
+
+      <.flash type="info" message="User created" />
+  """
+  def flash(assigns) do
+    ~H"""
+    <%= if is_binary(@message) and byte_size(String.trim(@message)) > 0 do %>
+      <p class={"alert alert-#{@type}"} role="alert"><%= @message %></p>
```

Comment:
```suggestion
      <p class={"alert alert-#{@type}"} role="alert">
        <%= @message %>
      </p>
```

That's how it would be formatted by the formatter :)

Reply:
yeah, good point 😄